frontend/src/views/SettingsView.vue

```diff
@@ -26,7 +26,7 @@
       <v-col>
         <!-- Users Table -->
         <v-skeleton-loader :loading="!usersAndFacilities" type="table-tbody">
-          <v-data-table :headers="headersUsers" :items="filteredUserFacilities" item-key="contactId" item-value="contactId" show-expand density="compact" :expanded.sync="expanded">
+          <v-data-table :headers="headersUsers" :items="sort(filteredUserFacilities)" item-key="contactId" item-value="contactId" show-expand density="compact" :expanded.sync="expanded">
```

Comment:
Would it make more sense to just do the sort in the computed filteredUserFacilities method? Also, in filteredUserFacilities it might make sense to return early (with sort) if there's no facilityNameFilter defined. I assume now it's checking for contains '' and the result ends up true which isn't necessarily intuitive.